### ISSUE TITLE
How use ASLayoutSpec for subnodes?

### ISSUE THREAD

**Issue by bimawa** [facebook/AsyncDisplayKit#731]
i create Custom header DisplayNode(with image and text subnodes) and setup layouts for subnodes in `layoutSpecThatFits:` but now i create custom ASCellNode with this header node and try call `measure` or `calculatedLayout`  for calculating headerNode with it subnodes, but nothing. What i wrong understand?

**Comment by bimawa** [comment]
I found my mistake sry.

**Comment by appleguy** [comment]
@bimawa not a worry, thanks for posting anyway!  Would love to answer any questions you have in the future, so don't hesitate to make another issue.

**Comment by bimawa** [comment]
@appleguy love u for that work with community!

**Comment by appleguy** [comment]
@bimawa :-D glad you're trying it out!  Have you seen the NSSpain talk yet?  I am making a slightly improved edit of it to post in the next day or two, so if you haven't seen it yet, wait until I post.  It covers the 2.0 stuff and has pretty useful information if you are using LayoutSpecs.

**Comment by bimawa** [comment]
No i see that: `NSLondon: Asynchronous UI by Scott Goodson - July 2014`. Cool i will be wait. Can u give link at NSSpain to me?

**Comment by bimawa** [comment]
I so much hate iOS SDK for micro bugs related with UITableView CollectionView and etc. And when i see ADK i understand.. I'm not alone anymore in this iOS dev world.

**Comment by bimawa** [comment]
@appleguy i found NSSpan conference, but i not found ur speak. Where can i get it?

**Comment by appleguy** [comment]
@bimawa haven't forgotten!  I have unfortunately not been able to post the talk yet, and am waiting on some materials from NSSpain.  I hope to post it by the end of Wednesday pacific time, but if I miss that for any reason, it won't be until at least Saturday.  All of Tuesday, Thursday, and Friday I have meetings at work and need to catch up with the day's email at night.

Attached is an early form of documentation for ASLayoutSpec that one of the most experienced iOS engineers at Pinterest is working on (@rcancro).  I'll post and let you know ASAP if the video or other documentation becomes available.  The main place to watch is the Releases section here, or a post in Paper Engineering Community (Facebook) and the Google Group (AsyncDisplayKit).

[ASLayoutSpecs 10-12-15.pdf](https://github.com/facebook/AsyncDisplayKit/files/12187/ASLayoutSpecs.10-12-15.pdf)

**Comment by bimawa** [comment]
@appleguy forgive my impatience, a strong desire to explore it all ... thanks for the file with him, I 'll wait just a little bit :)

**Comment by appleguy** [comment]
@bimawa Here's the edited video. It's a bit hacked up, but definitely less jarring to watch than the original. Please feel free to share it around!

Looking forward to any questions you have posted in new Issues.  Thank you for your patience, and interest in the project :).  I'm always glad to have kind and interesting people participate in the community.

https://www.youtube.com/watch?v=RY_X7l1g79Q

**Comment by bimawa** [comment]
@appleguy Thx a lot!!! I love you :)
